fl4health/clients/ditto_client.py

```diff
@@ -153,12 +132,15 @@ def set_parameters(self, parameters: NDArrays, config: Config, fitting_round: bo
             log(INFO, "Setting the global model weights")
             self.parameter_exchanger.pull_parameters(parameters, self.global_model, config)
 
+    def update_before_train(self, current_server_round: int) -> None:
```

Comment:
Moving the setting of the initial_global_tensors to the `update_before_train` function so that it isn't done during validation. It wasn't causing any issues, but it's unnecessary and this function conveniently isolates both the `set_parameters` functionality and  ensures that the tensors are saved only once before local training starts